Patterns/Turkish/Turkish-DateTime.yaml

```diff
@@ -123,7 +123,7 @@ DateUnitRegex: !simpleRegex
 DateTokenPrefix: ''
 TimeTokenPrefix: ''
 TokenBeforeDate: ''
```

Comment:
Can you add a comment here in the YAML file explaining why TokenBeforeDate needs to stay empty and TokenListBeforeDate populated for DateTimePeriod?

Reply:
Comment added.